core/compile.go

```diff
@@ -56,7 +56,7 @@ func CompileRecipe(recipePath string, runtime string, isRoot bool, origGid int,
 	return nil
 }
 
-// compileDocker compiles and builds a Docker image using the provided recipe
+// Build a Docker image using the specified recipe
```

Comment:
the docker and podman functions both build oci images, a better comment would be "Build an oci image using the specified recipe through docker"

Reply:
same thing for compilePodman

Reply:
resolved by ddd1e3af7cc494fc31064fa68f5eec14a6b82c24